<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelRegular">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 100, 100);
padding-left: 10;
color: rgb(255, 255, 155);</string>
          </property>
          <property name="text">
           <string>Using regular expressions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="regularLine">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(55, 255, 255);
border-color: rgb(81, 81, 81);</string>
          </property>
          <property name="maxLength">
           <number>12</number>
          </property>
          <property name="placeholderText">
           <string>Type the cell number here...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelRegularStatus">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 155, 255);</string>
          </property>
          <property name="text">
           <string>&lt; - Please enter the cell number in this field</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelMasked">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 100, 100);
padding-left: 10;
color: rgb(255, 255, 155);</string>
          </property>
          <property name="text">
           <string>Using masks</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="maskedLine">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(55, 255, 255);
border-color: rgb(81, 81, 81);</string>
          </property>
          <property name="inputMask">
           <string>+7 (999) 999-99-99</string>
          </property>
          <property name="maxLength">
           <number>18</number>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelMaskedStatus">
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 155, 255);</string>
          </property>
          <property name="text">
           <string>&lt; - Please enter the cell number in this field</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
